aggregator/src/task.rs

```diff
@@ -111,6 +111,8 @@ pub struct Task {
     collector_auth_tokens: Vec<AuthenticationToken>,
     /// HPKE configurations & private keys used by this aggregator to decrypt client reports.
     hpke_keys: HashMap<HpkeConfigId, (HpkeConfig, HpkePrivateKey)>,
+    /// Configuration option to add a length prefix for the public share in the input share AAD.
+    input_share_aad_tweak: bool,
```

Comment:
nit: perhaps name this something more descriptive, like `input_share_include_public_share_length_prefix` (though that's a little wordy). The doc comment makes it clear what this flag controls, but the field name itself is a little too generic IMO.